src/main/java/com/assertthat/selenium_shutterbug/core/Snapshot.java

```diff
@@ -176,7 +185,7 @@ public T withThumbnail(double scale) {
      * @return instance of type Snapshot
      */
     public T monochrome() {
-        this.image = ImageProcessor.convertToGrayAndWhite(this.image);
```

Comment:
Need to keep assignment to image so subsequent chained methods can use updated image.